vda5050_connector/vda5050_connector_py/vda5050_controller.py

```diff
@@ -665,7 +665,7 @@ def _update_state_from_adapter(self, order_state: VDAOrderState):
                 "distance_since_last_node": order_state.state.distance_since_last_node,
                 "battery_state": order_state.state.battery_state,
                 "errors": current_errors + order_state.state.errors,
-                "informations": order_state.state.informations
+                "information": order_state.state.information
```

Comment:
Hey @bekirbostanci for this to work the field must also be renamed in `vda5050_msgs/msg/OrderState.msg`, right ?

Reply:
yes you are right I changed in my workspace however I forgot it,

Reply:
I changed it also the message